daemon/algod/api/algod.oas2.json

```diff
@@ -3377,6 +3377,14 @@
           "items": {
             "type": "integer"
           }
+        },
+        "budget-added": {
+          "description": "Total budget added during execution of app calls in the transaction group.",
+          "type": "integer"
+        },
+        "budget-consumed": {
```

Comment:
What do you think about prefixing these new fields with `app-`, e.g. `app-budget-added`?

Since it probably makes sense to add a similar field for logicsigs in the future, I think this might help distringuish between them.

Reply:
Good point - I will try to push another commit to add logic sig costs later today